### ISSUE TITLE
wrong allocation in FL studio

### ISSUE THREAD

**Issue by wariorz135291** [cmajor-lang/cmajor#46]
Hi,
I'm just here to say thank you 1st.
But in Fl Studio, the Cmajor Gui is opening as a Mixing plugin, in which case we can use it as a VST and make melodies with.

Thanks

**Comment by julianstorer** [comment]
Yes, it's annoying that some hosts treat VSTs differently according to whether they say they're an instrument or not. We know about this, we'll figure out a plan such as having different builds which are hard-coded to be instruments/effects etc.

**Comment by wariorz135291** [comment]
Will wait for it. Just wanted to let you know.

**Comment by Dying-Soldier-15** [comment]
any progress yet?
